_posts/2018-03-22-industrialization-versus-data-selection.md

```diff
@@ -0,0 +1,64 @@
+# Brace yourself data selection, industrialization is coming!
+
+Industrialization is one of the most challenging problems for a start-up like ours. In fact, the research world has little to no knowledge when it comes to productivity. And this matter struck us when we thought about building language models (referred as LM in the following) massively, especially regarding data selection.
```

Comment:
This is a little bold statement no? Kaldi is coming from the research world, and we can say that it's __productive__ no? What do you mean exactly? That research optimizing time and cost is more an industry problem than a research one?